tests/checkup_test.go

```diff
@@ -42,7 +42,7 @@ const (
 	testConfigMapName                   = "dpdk-checkup-config"
 	testCheckupJobName                  = "dpdk-checkup"
 
-	testTimeout = 10 * time.Minute
+	testTimeout = 11 * time.Hour
```

Comment:
What is the reason to move from 10 minutes to eleven hours?

Reply:
don't not need to be there. DONE